src/v2/view-builder/ViewFactory.js

```diff
@@ -49,6 +51,9 @@ const VIEWS_MAPPING = {
   'select-factor-enroll': {
     [DEFAULT]: SelectFactorEnrollView,
   },
+  'skip': {
```

Comment:
looks like in the optional enroll case, we are still rendering `select-authenticator-enroll` but not `skip` form hence doesn't need this config?

Reply:
Right, this is not used. will remove. This got added when I was testing just with skip form.